test/Microsoft.ML.FSharp.Tests/Microsoft.ML.FSharp.Tests.fsproj

```diff
@@ -0,0 +1,50 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Unix'">$(TargetFrameworks); net461</TargetFrameworks>
+    <NoWarn>2003;$(NoWarn)</NoWarn>
+    <PublicSign>false</PublicSign>
+    <SourceLink></SourceLink>
```

Comment:
What's this for?  Is this necessary?

Reply:
I just got a build error when building from the IDE using VS 15.7 because for .NET 4.6.1 compilation "full" PDB symbols (not portable) were being produced), I can reproduce if I build with `msbuild` not `dotnet`.  Full PDB are not compatible with sourcelink. It doesn't go away even if you set DebugType to "portable".  

I think it is an F# build targets bug fixed in 15.8.   Anyway for now seemed sensible to turn of SourceLink for the test project